recipe/build.sh

```diff
@@ -3,3 +3,5 @@
 python setup.py install --single-version-externally-managed --record=record.txt
 
 cp bdist_conda.py "${PREFIX}/lib/python${PY_VER}/distutils/command/"
+
+rm -f "${PREFIX}/bin/conda"
```

Comment:
This certainly doesn't hurt, and I understand why you're doing it, but we should cut it at some point in the future.